src/server/models/match.model.js

```diff
@@ -0,0 +1,32 @@
+'use strict';
+
+import mongoose from 'mongoose';
+
+const Schema = mongoose.Schema;
+let matchSchema = new Schema({
+  mentorId: String,
```

Comment:
I was about to divide user collection and match collection.
match collection is made up with string,
but using user object seems good.

I'll reflect that